skunkworks_crow/src/main/res/values/strings.xml

```diff
@@ -13,6 +13,8 @@
     <string name="send_selected">Send</string>
     <string name="select_all">Select all</string>
     <string name="clear_all">Clear all</string>
+    <string name="send_count">Send(</string>
+    <string name="closed_bracket">)</string>
```

Comment:
You can remove the above two lines and can replace it with
`<string name="send_count">Send (%d)</string>`
and then can use `sendButton.setText(getString(R.string.send_count),selectedForms.size());`

Reply:
if I am using like that then the android studio gives cannot resolve method settext( java.lang.string,int)

Reply:
I think in C we can do like that taking int as %d

Reply:
> You can remove the above two lines and can replace it with
> `<string name="send_count">Send (%d)</string>`
> and then can use `sendButton.setText(getString(R.string.send_count),selectedForms.size());`

Instead of that I use 

* sendButton.setText(String.format(getString(R.string.send_count), selectedForms.size()));

Reply:
You can use parameters in strings.xml 
https://stackoverflow.com/questions/2397613/are-parameters-in-strings-xml-possible

Reply:
> You can remove the above two lines and can replace it with
> `<string name="send_count">Send (%d)</string>`
> and then can use `sendButton.setText(getString(R.string.send_count),selectedForms.size());`

have you used exactly the same method, because it is working fine in my system without errors.

Reply:
@lakshyagupta21 yeah I used the same method when used the format it looks fine as we expected